mdoc/mdoc.Test/mdoc.Test.csproj

```diff
@@ -37,10 +37,14 @@
       <HintPath>..\..\external\Windows\Windows.Foundation.UniversalApiContract.winmd</HintPath>
     </Reference>
   </ItemGroup>
+
   <ItemGroup>
     <PackageReference Include="Mono.Cecil" Version="$(NuGetVersionMonoCecil)" />
     <PackageReference Include="NUnit" Version="$(NuGetVersionNUnit)" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.2.0" />

```

Comment:
should we define the version string in `Directory.Build.props`?

Reply:
I moved these two package versions to Directory.Build.props.